plugin/ocgrpc/server.go

```diff
@@ -36,12 +31,20 @@ type ServerHandler struct {
 	// NoStats may be set to disable recording OpenCensus Stats around each
 	// gRPC method.
 	NoStats bool
+
+	// NoReadTraceContext may be set to true to avoid reading trace metadata
+	// from inbound requests.
+	//
+	// You should set this if your gRPC server is a public-facing service and
+	// your load balancer does not strip metadata. Otherwise, a malicious user
+	// will be able to trigger traces in your backend.
+	NoReadTraceContext bool
```

Comment:
NoTraceContextReads?

Reply:
That makes it sounds like "there are not trace context reads" where what I want to say is "don't read the trace context"

Reply:
Gotcha gotcha, ACK.